pkgs/applications/misc/remarkable/rmview/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, python3Packages, wrapQtAppsHook }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "rmview";
+  version = "2.1";
+
+  src = fetchFromGitHub {
+    owner = "bordaigorl";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0zyngirpg808k1pkyhrk43qr3i8ilvfci0wzwk4b5f6f9cmjs7kb";
+  };
+
+  nativeBuildInputs = with python3Packages; [ pyqt5 wrapQtAppsHook ];
+  propagatedBuildInputs = with python3Packages; [ pyqt5 paramiko twisted ];
+
+  preBuild = ''
+    pyrcc5 -o src/rmview/resources.py resources.qrc
+  '';
+
+  preFixup = ''
+    wrapQtApp "$out/bin/rmview"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A fast live viewer for reMarkable 1 and 2";
+    homepage = "https://github.com/bordaigorl/rmview";
+    license = licenses.gpl3;
```

Comment:
Plus or only?

Reply:
Actually not sure, seems like the author just put the default GPL3 `LICENSE` file in the repo and nothing anywhere in the sources.
https://github.com/bordaigorl/rmview/blob/vnc/LICENSE

Reply:
It does seem `licenses.gpl3Plus`, because otherwise the author needs to explicitly remove "any or later" references.

Reply:
@thiagokokada I don't think that is true:

> Each version is given a distinguishing version number. If the Program specifies that a certain numbered version of the GNU General Public License “or any later version” applies to it, you have the option of following the terms and conditions either of that numbered version or of any later version published by the Free Software Foundation. If the Program does not specify a version number of the GNU General Public License, you may choose any version ever published by the Free Software Foundation.

So, it's only "or any later version" if: (1) the author explicitly specifies it's GPLv3 or later; or (2) if the author does not specify a version of the GPL at all.

The README states at the end `GPLv3`, so that's a specific version and it's thus `gpl3Only`, unless the source files have headers stating something else of course.

Reply:
it is gpl3only.